app/controllers/api/v0/podcast_episodes_controller.rb

```diff
@@ -13,13 +13,14 @@ def index
         @page = params[:page]
 
         if params[:username]
-          @podcast = Podcast.find_by(slug: params[:username]) || not_found
+          @podcast = Podcast.reachable.find_by(slug: params[:username]) || not_found
```

Comment:
I think you should just use `find_by!`, `not_found` just does the same thing: raises the active record not found exception